swap_meet/item.py

```diff
@@ -1,2 +1,25 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category = "", condition = 0):
+        self.category = category
+        self.condition = condition 
+
+    def get_by_category(self):
+        pass
+   
+    def __str__(self):
+        return f"Hello World!"
+
+    def condition_description(self, condition):
+        for clothes in range(0, 6):
```

Comment:
This loop isn't needed.  The `clothes` variable isn't used inside the loop, and the conditionals will result in returning on the first iteration.